rust/processor/src/models/token_v2_models/v2_token_utils.rs

```diff
@@ -412,27 +495,71 @@ impl PropertyMapModel {
     }
 }
 
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub struct AggregatorSnapshot<T> {
+    // TODO: #[serde(deserialize_with = "deserialize_from_string")] is showing error
+    value: T,
+}
```

Comment:
wait so does this actually work?

Reply:
Is the `serde` annotation required for the code to work?